pkgs/tools/misc/broot/default.nix

```diff
@@ -7,30 +7,36 @@
 , libiconv
 , zlib
 , Security
+, oniguruma
+, pkg-config
+, libgit2
```

Comment:
Would be nice if this could be kept in order: lib -> stdenv -> library functions -> nativeBuildInputs -> buildInputs.